teams/vsa/engineering/forms-system/custom_component_simple.md

```diff
@@ -0,0 +1,147 @@
+# Updating Form State with an onChange Handler Within a Custom React Component
+
+Based on answers from *Christopher Valarida* from Slack discussions:
+
+- **A field can not trigger a change to another field** \
+*We can! There's an ***updateFormData** available. Put that sucker in the **uiSchema** on a field and changes to it will trigger the callback you pass it. I highly recommend thinking hard about whether you need it first, however. It's a known footgun; if you're not careful, you can make a lot of sneaky bugs for your future self.*
```

Comment:
@bradl3yC this wasn't necessary nor sufficient for the complex scenario, right? (Due to "another field" being "another field in another react component?)

Reply:
@short000 we didn't know about the existence of this function until after we had completed our work - I don't think it would work for our application, though as we wanted to update the state of another component